index.html

```diff
@@ -5,9 +5,45 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <title>My Portfolio</title>
   </head>
   <body>
-    <h1>Billy's Portfolio</h1>
+    <div class="container">
+      <div class="toolbar">
+        <p class="toolbar-text">John Doe</p>
+        <a href="#" class="menu"><img src="Assets/ham.png" alt="" /></a>
+      </div>
+      <div class="headline">
+        <h1 class="content1">Hey There.<br />I’m Alex</h1>
+        <h2 class="content2">I’am a Software Developer</h2>
+        <p class="content3">
+          I can help you build a product , feature or<br />website Look through
+          some of my work<br />and experience! If you like what you see<br />and
+          have a project you need coded,<br />don’t hestiate to contact me.
+        </p>
+        <div class="icons">
+          <a href="#"><img src="assets/hub.png" alt="" /></a>
+          <a href="#"><img src="assets/in.png" alt="" /></a>
+          <a href="#"><img src="assets/hnd.png" alt="" /></a>
+          <a href="#"><img src="assets/tw.png" alt="" /></a>
+          <a href="#"><img src="assets/m.png" alt="" /></a>
```

Comment:
- [ ] These social media icons should be wrapped in list tag `<ul>` where each social media icon is a list item `<li>` This will make it more meaningful 😉